Source/BugsnagSessionTracker.m

```diff
@@ -172,4 +172,20 @@ - (void)handleHandledErrorEvent {
     }
 }
 
+- (void)handleUnhandledErrorEvent {
+    BugsnagSession *session = [self runningSession];
+
+    if (session == nil) {
+        return;
+    }
+
+    @synchronized (session) {
+        session.unhandledCount++;
+        if (self.callback && (self.config.shouldAutoCaptureSessions || !session.autoCaptured)) {
```

Comment:
is this logic correct? can this be extracted into a method? was hoping this could be simple:
```
if (self.callback) {
    self.callback(session);
}
```

Reply:
yeah, though it converts into a no-op. I simplified the logic instead!